### ISSUE TITLE
[Q] How to use this algorithm in real-time/on-line?

### ISSUE THREAD

**Issue by N1h1l1sT** [twitter/AnomalyDetection#91]
So, for off-line usage, you have the dataset, you put a date/time as the 1st column in a data.frame and a numeric one as the second, and you feed it to AnomalyDetectionTs()

This way you can get an off-line graph via $plot (not being re-rendered as new data come) and the actual anomalous entries on $anoms

But what if you wish to use it not for offline data, but for streaming data?
Can you get a responsive graph? and most importantly, can you get the increasing number of anomalies?

Or is the idea to merge the data into a big dataset and run it every X minutes/hours/etc with an "only_last" value?

**Comment by tharejamudit** [comment]
thinking of this

**Comment by N1h1l1sT** [comment]
Yeah, having an on-line way of getting anomalies and graph is very important..

**Comment by irisava** [comment]
Except for updating the data to contain the last streaming data every minutes/hours, is there any other method?
